api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageManager.java

```diff
@@ -1031,6 +1031,10 @@ private AbstractDataStorage updateDataStorageObject(AbstractDataStorage dataStor
                 storageProviderManager.getStorageProvider(dataStorage).getDefaultMountOptions(dataStorage));
         }
 
+        if (dataStorageVO.getMountDisabled() != null) {
```

Comment:
Maybe use the same logic as in `AbstractDataStorageFactory` with `BooleanUtils` to prevent null values

Reply:
I believe here we need explicitly check for null, because there could be a case, when storage is updated and this value is not provided, it means that we don't need to change current value. But if we will use here BooleanUtils.isTrue, we will not be able to update value from True to False for example.